### PR TITLE
Update repo to `SCIsegV2`

### PR DESCRIPTION
**Pull request by valosekj**
This PR updates the repo's README to SCIsegV2; see individual commit messages for details. I've also added a few comments on which I'd like your opinion @naga-karthik.

To see the rendered README: https://github.com/ivadomed/model_seg_sci/blob/jv/update_repo_to_SCIsegV2/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # SCISeg: Automatic Segmentation of T2-weighted Intramedullary Lesions in Spinal Cord Injury
 
-[![medRxiv](https://img.shields.io/badge/medRxiv-10.1101/2024.01.03.24300794v2-blue.svg)](https://www.medrxiv.org/content/10.1101/2024.01.03.24300794v2.full.pdf)
+[![medRxiv](https://img.shields.io/badge/medRxiv-10.1101/2024.01.03.24300794v2-blue.svg)](https://www.medrxiv.org/content/10.1101/2024.01.03.24300794v2.full.pdf) [![arXiv](https://img.shields.io/badge/arXiv-2407.17265-b31b1b.svg)](https://doi.org/10.48550/arXiv.2407.17265)
```
Comment (valosekj) [inline README.md:3]: Or do we want to add `SCIsegV1` and `SCIsegV2` next to the badges?

```suggestion
`SCIsegV1`: [![medRxiv](https://img.shields.io/badge/medRxiv-10.1101/2024.01.03.24300794v2-blue.svg)](https://www.medrxiv.org/content/10.1101/2024.01.03.24300794v2.full.pdf) `SCIsegV2`: [![arXiv](https://img.shields.io/badge/arXiv-2407.17265-b31b1b.svg)](https://doi.org/10.48550/arXiv.2407.17265)
```

File: README.md [hunk 2]
```diff
@@ -78,5 +77,17 @@ If you find this work and/or code useful for your research, please cite our pape
...
 
 ```
+@article {karthik2024scisegv2universaltoolsegmentation,
+      title={SCIsegV2: A Universal Tool for Segmentation of Intramedullary Lesions in Spinal Cord Injury}, 
+      author={Enamundram Naga Karthik* and Jan Valošek* and Lynn Farner and Dario Pfyffer and Simon Schading-Sassenhausen and Anna Lebret and Gergely David and Andrew C. Smith and Kenneth A. Weber II au2 and Maryam Seif and RHSCIR Network Imaging Group and Patrick Freund and Julien Cohen-Adad},
+      year={2024},
+      eprint={2407.17265},
+      archivePrefix={arXiv},
+      primaryClass={cs.CV},
+      url={https://arxiv.org/abs/2407.17265},
```
Comment (valosekj) [inline README.md:90]: I got this ref from [arXiv](https://arxiv.org/abs/2407.17265) by clicking on "Export BibTeX Citation" button. But the fields are now different from SCIsegV1 ref. Maybe we could unify them, right?

File: README.md [hunk 3]
```diff
@@ -13,18 +12,18 @@ The model was trained on raw T2-weighted images of SCI patients from seven sites
 
 ## Updates
 
-### 2024-XX-XX
+### 2024-07-24
 
-* We have released **SCIsegV2: A Universal Model for Intramedually Lesion Segmentation in SCI**. The new model is trained on a larger cohort covering both traumatic and non-traumatic SCI lesions. The new model is available for download using the same installation instructions as below.
-* The computation of midsagittal tissue bridges is now fully-automatied and powered by SCIsegV2. 
+* We have released **SCIsegV2: A Universal Model for Intramedullary Lesion Segmentation in SCI**. The new model is trained on a larger cohort covering both traumatic and non-traumatic SCI lesions. SCIsegV2 is available as part of the SCT via the `sct_deepseg` function; see the installation instructions below.
+* The computation of midsagittal tissue bridges is now fully-automated and powered by SCIsegV2. The automatic computation of tissue bridges is available via the `sct_analyze_lesion` function as part of SCT v6.4 and higher.
```
Comment (valosekj) [inline README.md:18]: I'm thinking whether we want to add Fig2 from https://arxiv.org/pdf/2407.17265 here to illustrate the tissue bridges.

File: README.md (hunk 1 above)
Comment (naga-karthik) [reply README.md:3]: Yeah I think adding SCIsegV1 and SCIsegV2 next to the badges would be good!

File: README.md (hunk 3 above)
Comment (naga-karthik) [reply README.md:20]: sure! I don't have the figure so I couldn't add it

File: README.md (hunk 2 above)
Comment (naga-karthik) [reply README.md]: I think it's okay, don't know what to remove, it's a copy-able block so it should be fine

File: README.md (hunk 3 above)
Comment (valosekj) [reply README.md:20]: Okay! The figure added in https://github.com/ivadomed/model_seg_sci/commit/ec09d39dedadf4cd4a0295d5d3f08b45f533fe6b

File: README.md (hunk 1 above)
Comment (valosekj) [reply README.md]: Thanks for adding the text.

It seems that markdown formatting does not center the text to the badge:

Screenshot 2024-07-26 at 7 36 00 AM

Maybe we could add a new line?

Screenshot 2024-07-26 at 7 35 51 AM

**Comment by valosekj** [comment]
Thanks for your commits, @naga-karthik! Once we resolve [this comment](https://github.com/ivadomed/model_seg_sci/pull/92#discussion_r1692938560), we can merge.

File: README.md (hunk 1 above)
Comment (naga-karthik) [reply README.md]: https://github.com/ivadomed/model_seg_sci/pull/92/commits/8948b619a9c8359e2f5ef37722e7cccf12c5e3bb found something even better!

**Comment by naga-karthik** [comment]
please merge once you have resolved it!

File: README.md (hunk 1 above)
Comment (naga-karthik) [reply README.md]: Screenshot 2024-07-26 at 4 03 23 PM

Comment (valosekj) [reply README.md]: nice! merging!

**Pull request merged**